packages/transport/src/transports/nodeusb.ts

```diff
@@ -0,0 +1,40 @@
+import { WebUSB } from 'usb';
+
+import { Transport } from './abstract';
+import { UsbTransport } from './usb';
+import { SessionsClient } from '../sessions/client';
+import { SessionsBackground } from '../sessions/background';
+import { TransportUsbInterface } from '../interfaces/usb';
+
+// notes:
+// to make it work I needed to run `sudo chmod -R 777 /dev/bus/usb/`
```

Comment:
Interesting, that makes sense, since the browser can manage the permisions but without browser in a linux system the user has to grant access to usb port. This is not nice for user, since many users are not going to be happy to do that.

If this is This could be a VERY interesting point to use webusb from electron since I assume the permissions would be granted in a similar way as in the browser.

Reply:
nodeusb moved to another PR